openpype/modules/deadline/plugins/publish/submit_houdini_render_deadline.py

```diff
@@ -45,7 +45,10 @@ class VrayRenderPluginInfo():
 @attr.s
 class RedshiftRenderPluginInfo():
     SceneFile = attr.ib(default=None)
-    Version = attr.ib(default=None)
+    # "1" was selected as the default RS version as it's the default
+    #  version in RS deadline plugin.
+    # https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/app-redshift.html#plug-in-configuration
```

Comment:
Let's remove the doc link - it does show `1` being listed but it's not super explicit about that being the 'backend' case as well. Plus, the link can easily grow "outdated" because it links to a Deadline version.
```suggestion
```